.gitlab-ci/fedora.Dockerfile

```diff
@@ -0,0 +1,59 @@
+FROM registry.fedoraproject.org/fedora:33
+
+RUN dnf -y install \
+    ccache \
+    desktop-file-utils \
+    gcc \
+    gettext \
+    git \
+    gobject-introspection-devel \
+    gtk-doc \
+    ibus-devel \
+    intltool \
+    krb5-devel \
+    libpwquality-devel \
+    libsecret-devel \
+    meson \
+    ninja-build \
+    "pkgconfig(accountsservice)" \
+    "pkgconfig(cheese)" \
+    "pkgconfig(cheese-gtk)" \
+    "pkgconfig(flatpak)" \
```

Comment:
Looks like this was added upstream as part of the parental controls feature (9cd679a268e5627fb5d9abdf3e5eca9812d9fce4), but then the build dependency on `libflatpak-dev` was not kept during upstreaming so I guess it's not needed and this isn't either?

Reply:
libflatpak-dev is a build dependency of malcontent, which is compiled later down this dockerfile.